packages/dart/sshnp_flutter/macos/Runner/Info.plist

```diff
@@ -22,54 +22,56 @@
 		<string>$(FLUTTER_BUILD_NUMBER)</string>
 		<key>LSMinimumSystemVersion</key>
 		<string>$(MACOSX_DEPLOYMENT_TARGET)</string>
+		<key>NSFaceIDUsageDescription</key>
+		<string>$(PRODUCT_NAME) requires Touch Id or Face ID permission to authenticate with the service</string>
```

Comment:
Do we require this still? I don't think we do.

Reply:
Good catch, I'm not using biometric storage anymore so It should be removed.